plugins/UserLanguage/UserLanguage.php

```diff
@@ -23,7 +23,7 @@ public function postLoad()
 
     public static function footerUserCountry(&$out)
     {
-        $out .= '<h2 piwik-enriched-headline>' . Piwik::translate('UserLanguage_BrowserLanguage') . '</h2>';
+        $out .= '<h2 vue-entry="CoreHome.EnrichedHeadline">' . Piwik::translate('UserLanguage_BrowserLanguage') . '</h2>';
         $out .= FrontController::getInstance()->fetchDispatch('UserLanguage', 'getLanguage');
     }
```

Comment:
Actually it seems everything in this file is obsolete. The event `Template.footerUserCountry` used to exist in the [UserCountry plugin](https://github.com/matomo-org/matomo/blob/0dd0926c012f54d9a2a43249e0d11ff38394cdc7/plugins/UserCountry/templates/index.twig#L15). But it had been removed a long time ago. Would be nice if you could simply update the PR, to remove the methods in this class.